### DISCUSSION TITLE
Plot code querry

### DISCUSSION THREAD

**Discussion by saleharashid01** [AammarTufail/python_ka_chilla2023#57] [Q&A]
![s](https://user-images.githubusercontent.com/121869148/211893638-b75be555-74fe-470d-816e-d681ed7612fa.jpg)
@[22:10](https://www.youtube.com/watch?v=IVtFDKR_KOA&t=1330s), you use a code
sns.FacetGrid(kashti, hue='survived', col="pclass", margin_titles=True,

                palette={1:'pink',0:'green'})
g= g.map(plt.scatter,'fare','age',edge_color='w').add_legend();

**here g=g.map he, what g. symbolizes?, when i run it, they are saying " g is not defined**

**Comment by Muhammad-Ali-Butt** [comment]
sns.facetgrid wali line ko g mai pehlay assign karo 

g=sns.FacetGrid(kashti, hue='survived', col="pclass", margin_titles=True,  **g is a variable jis mai canvas draw kia gia .facetgrid() use kr k**

            palette={1:'pink',0:'green'})

then write g= g.map(plt.scatter,'fare','age',edgecolor='w').add_legend(); **now in this the canvas which is in g is being mapped i.e. 
                                                                                                                         being plotted as a scatterplot with fare in x axis and age in y**

**Reply by Atifidrees2023** [reply]
g=g.map(plt.scatter,"fare", "age", edgecolor="w").add_legend();
Check the code for edgecolor please

**Reply by Muhammad-Ali-Butt** [reply]
@Atifidrees2023 ty for pointing that out bhai i just copied her code from above without paying attention to spelling / syntax etc
